source/solvers/postprocessing_velocities.cc

```diff
@@ -40,14 +40,35 @@ AverageVelocities<dim, VectorType, DofsType>::calculate_average_velocities(
   // Get the inverse of the total time with the first time step since
   // the weighted velocities are calculated with the first velocity when
   // total time = 0.
-  inv_range_time = 1. / ((current_time - real_initial_time) + dt_0);
+  total_time_for_average = (current_time - real_initial_time) + dt_0;
+  inv_range_time         = 1. / total_time_for_average;
 
   // Calculate the average velocities.
   average_velocities.equ(inv_range_time, sum_velocity_dt);
 
   this->calculate_reynolds_stresses(local_evaluation_point);
 }
 
+
+template <int dim, typename VectorType, typename DofsType>
+void
+AverageVelocities<dim, VectorType, DofsType>::update_average_velocities()
+{
+  // Get the inverse of the total time with the first time step since
+  // the weighted velocities are calculated with the first velocity when
+  // total time = 0.
```

Comment:
Sorry I read it multiple times and I do not get what you are saying there ^^" 

I get the part "the weighted velocities are calculated with the first velocity when total time = 0." (though I am not sure where to see that in the code?), but what does "Get the inverse of the total time with the first time step" mean? Is the part "with the first time step" really part of the comment or should it be removed?

Reply:
Agreed, I have also no clue what is meant by that comment here.